cmd/osbuild-worker/jobimpl-osbuild-koji.go

```diff
@@ -61,6 +61,8 @@ func validateKojiResult(result *worker.OSBuildKojiJobResult, jobID string) {
 		reason := "osbuild job was unsuccessful"
 		logWithId.Errorf("osbuild job failed: %s", reason)
 		result.JobError = clienterrors.WorkerClientError(clienterrors.ErrorBuildJob, reason)
+	} else {
+		logWithId.Infof("koji-build job succeeded")
```

Comment:
```suggestion
		logWithId.Infof("osbuild-koji job succeeded")
```

would be slightly better :thinking: